uint8 gps_enabled
string reason
---